### PR TITLE
Web Dev Lesson -- Tailwind Assignment

### PR DESCRIPTION
**Pull request by LuckyHariharan**
Hi Raveen, I put up a simple assignment for students to lock-in their tailwind css knowledge. Regarding the   `const assignmentId = "2cf9156a-4f6f-452d-b09a-2c54f19a7b40";` , how do I create an id for this assignment? 

Ty in advance for any feedback!
image

### CODE REVIEW

**Review by vithushan19 — APPROVED** [review]

**Review by vithushan19 — APPROVED** [review]
this looks great nice work!

File: pages/studentPortal/web/React/assignments/tailwindAssignment.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        "Please submit your assignment by pasting a link in the box below.",
+      placeholder: "Assignment link goes here",
+      assignmentId: assignmentId,
+      link: "",
+    },
+    {
+      component: "hint-list",
+      hintRow: [
+        {
+          description: "Setup TailwindCSS using Play CDN, follow step one",
```
Comment (iamnotreddy) [inline pages/studentPortal/web/React/assignments/tailwindAssignment.tsx:142]: let's abstract the tailwind setup from this assignment,

you can fork this [template](https://codesandbox.io/s/react-typescript-tailwind-template-6t0ywc) then setup the assignment:

File: pages/studentPortal/web/React/assignments/tailwindAssignment.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        ) : null}
+        <div className="flex my-8 mr-8 sm:justify-end">
+          <Button onClick={handleContinue} label="Continue" />
+        </div>
+      </div>
+    </>
+  );
+};
+export async function getServerSideProps() {
+  const assignmentId = "2cf9156a-4f6f-452d-b09a-2c54f19a7b40";
```
Comment (iamnotreddy) [inline pages/studentPortal/web/React/assignments/tailwindAssignment.tsx:109]: this assignmentId is the same as the other assignment

create a new assignment in the coding_assignments table and name it Tailwind Assignment. You can then paste the ID that hasura generates over here

**Review by iamnotreddy — CHANGES_REQUESTED** [review]
Nice work on this assignment. The grid vs. flex explanation was much clearer than the first time. Great interactive exercise for students to learn tailwind.

Couple things before approving:
- make sure you generate a new assignment ID in hasura and paste it in the server side props
- you're missing a template section. Fork this [template](https://codesandbox.io/s/react-typescript-tailwind-template-6t0ywc) to create a starting point for the assignment. Maybe you want to have the student start with unstyled text and / or other HTML elements so it's clearer to them that they need to restyle them using tailwind. Once you save this link, you can put it inside the template sub-component

**Review by iamnotreddy — APPROVED** [review]
nice

**Pull request merged**